float32 ch1_value
float32 ch2_value
string ch1_unit
string ch2_unit
